PYME/Analysis/BleachProfile/fitRecover.py

```diff
@@ -24,37 +24,37 @@
 from . import kinModels
 
 from PYME.Analysis._fithelpers import *
-
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
I still prefer deferring the pyplot import to the actual function call, as it makes things much more robust if and when you need to specify a matplotlib backend. Matplotlib chooses a backend the first time you import pyplot if you haven't already specified one, meaning that you have to be super careful about the order you import modules if you import pyplot (or pylab) at the module level.

Deferring the pyplot import until function execution time also means that you can do a lot more on a headless system (for example, a module level pyplot import can prevent test coverage from working on a headless system).

I know this is not currently consistent throughout the code, but I'm working towards it.

Reply:
Oops! Sorry David - I knew you'd prefer that for matplotlib and actually just moved everything to the function since there's only one function in that file, but forgot to push it.